.github/workflows/c-cpp.yml

```diff
@@ -74,3 +87,25 @@ jobs:
         with:
           name: vita3k-${{ steps.git_short_sha.outputs.value }}-${{ matrix.os }}
           path: build/bin
+
+      - name: Zip Artifacts
+        uses: papeloto/action-zip@5f1c4aa
+        with:
+          files: build/bin
+          dest: ${{ matrix.os }}.zip
+        if: github.ref == 'refs/heads/master'
+
+      - name: Update the CI tag
+        uses: Xotl/cool-github-releases@v1
+        with:
+          mode: update
+          isPrerelease: true
+          tag_name: continous
+          release_name: "Automatic CI builds"
+          body_mrkdwn: |
+            TODO: fill this with something note worthy
```

Comment:
Can you just remove this? Ill just create an issue for this.

Reply:
Just forgot about it, in theory just having the last successfully commit for reference in there would be enough. If someone is bored enough, they can add stuff like commit message.